framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -1359,13 +1359,25 @@ public Void visitUnary(UnaryTree node, Void p) {
                 || (node.getKind() == Tree.Kind.POSTFIX_DECREMENT)
                 || (node.getKind() == Tree.Kind.POSTFIX_INCREMENT)) {
             AnnotatedTypeMirror varType = atypeFactory.getAnnotatedTypeLhs(node.getExpression());
-            // For postfix increments/decrements, the value type is incorrect due to the workaround
-            // in GenericAnnotatedTypeFactory.addComputedTypeAnnotations(Tree, AnnotatedTypeMirror, boolean)
-            // for the following bug:
-            // See Issue 867: https://github.com/typetools/checker-framework/issues/867
-            // This means could result in a false warning (false positive) in some cases and a lack
-            // of a warning in other cases (false negative).
-            AnnotatedTypeMirror valueType = atypeFactory.getAnnotatedType(node);
+            Tree newValue = node;
```

Comment:
Why introduce this variable? The name is also rather confusing, given that it is a `Tree`, but named `newValue`. There is a `value` local variable further down, which is not related at all.
Can't you just use `node`?